proxy/Cargo.toml

```diff
@@ -11,6 +11,7 @@ workspace = true
 workspace = true
 
 [dependencies]
+analytic_engine = { workspace = true }
```

Comment:
Remove this dep, move parse_duration to where ReadableDuration is.

Reply:
Now the relevant constants and functions have been put into the common